plugin/api.go

```diff
@@ -48,6 +48,16 @@ type API interface {
 	// UpdateUser updates a user.
 	UpdateUser(user *model.User) (*model.User, *model.AppError)
 
+	// GetUserStatus will get a user's status.
+	GetUserStatus(userId string) (*model.Status, *model.AppError)
+
+	// GetUserStatusesByIds will return a list of user statuses based on the provided slice of user IDs.
+	GetUserStatusesByIds(userIds []string) ([]*model.Status, *model.AppError)
+
+	// UpdateUserStatus will set a user's status until the user, or another integration/plugin, sets it back to online.
+	// The status parameter can be: "online", "away", "dnd", or "offline".
```

Comment:
Should we expose these as `const`s? Also, should the parameter order by `userId, status` for consistency with other APIs?

Reply:
They are exposed as constants in the model directory

And sure I can flip the order to match the other APIs

Reply:
Hah, yes so they are. 👍